nuv/wsk_properties.go

```diff
@@ -53,3 +53,21 @@ func writeFileToNuvolarisHomedir(filename string, content []byte) (string, error
 	}
 	return path, nil
 }
+
+func getNuvolarisHomedir() (string, error) {
```

Comment:
This could be moved into util.go since it seems quite useful, what do you think?